File: exo3/main.php
<?php
class Human
{
    public $firstname;
    public $lastname;
    public $birthdate;

    function __construct($firstname, $lastname, $birthdate)
    {
        $this->firstname = $firstname;
        $this->lastname = $lastname;
        $this->birthdate = $birthdate;
    }
}
$new_human = new Human("Jean", "Bon", 20 / 02 / 2020);
echo ($new_human);

File: exo3/main.java
class Main {
    public static void main(String[] args) {
        newHuman1 = new Human("Jean", "Bon", "20/02/2020");
        System.out.print(newHuman1.firstName, newHuman1.lastName, newHuman1.birthDate);
    }
}

class Human {
    public String firstName;
    public String lastName;
    public String birthDate;

    Human(String firstName, String lastName, String birthDate){
        this.firstName = firstname;
        this.lastName = lastName;
        this.birthDate = birthDate
    }
}